tasty-plutus/src/Test/Tasty/Plutus/TestData.hs

```diff
@@ -71,10 +82,10 @@ data TestData (p :: Purpose) where
     --
     -- @since 3.0
     redeemer ->
-    -- | The tokens to be minted by the script.
+    -- | List of tokens to be minted by the script.
     --
     -- @since 4.1
-    Tokens ->
+    NonEmpty MintingPolicyTask ->
```

Comment:
The `since` should match when this changed, which I think is the current version?

Reply:
Yes, this is a change from 6.0. Fixed it.